WordPress/Classes/ViewRelated/Comments/WPStyleGuide+Comments.swift

```diff
@@ -13,50 +13,50 @@ extension WPStyleGuide {
         }
 
         public static func separatorsColor(isApproved approved: Bool) -> UIColor {
-            return approved ? WPStyleGuide.readGrey() : WPStyleGuide.alertYellowDark()
+            return approved ? .divider : UIColor(light: .warning(.shade90), dark: .warning(.shade40))
         }
 
         public static func detailsRegularStyle(isApproved approved: Bool) -> [NSAttributedString.Key: Any] {
-            let color = approved ? WPStyleGuide.littleEddieGrey() : WPStyleGuide.alertYellowDark()
+            let color = approved ? .textSubtle : UIColor(light: .warning(.shade90), dark: .warning(.shade40))
 
             return  [.paragraphStyle: titleParagraph,
                      .font: titleRegularFont,
                      .foregroundColor: color ]
         }
 
         public static func detailsRegularRedStyle(isApproved approved: Bool) -> [NSAttributedString.Key: Any] {
-            let color = approved ? WPStyleGuide.littleEddieGrey() : .errorDark
+            let color = approved ? .text : UIColor(light: .error(.shade90), dark: .error(.shade40))
```

Comment:
We've used `UIColor(light: .error(.shade90), dark: .error(.shade40))` in a lot of places here – should we give it a name and define it in one place?

Reply:
Same for warning